recipes/pyclipper/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pyclipper" %}
+{% set version = "1.1.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 8a8b6018d53fcce291f78dedca19994f82695eed3a2c9eff275691d4ed9aab51
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - setuptools_scm_git_archive
+    - setuptools_scm
+
+  run:
+    - python
+    - setuptools
+
+test:
+  commands:
+    - python -c "import pkg_resources; pkg_resources.require('pyclipper')"
+  imports:
+    - pyclipper
+
+about:
+  home: https://github.com/fonttools/pyclipper
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Cython wrapper for the C++ translation of the Angus Johnson's Clipper library (ver. 6.4.2)"
```

Comment:
wait, why doesn't this depend on cython?

Reply:
https://github.com/fonttools/pyclipper/blob/master/setup.py#L9